docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.7'
 services:
     postgres:
-        image: postgres:11
+        image: postgres:12
```

Comment:
If the postgres image is updated by a major increment, the existing data and database will need some migration. This seems to be a default behavior in Postgres. Because of this, the volumes-configuration could include the postgresql version like 

```
        volumes:
          - pgdata:/var/lib/postgresql@12/data
```

This will create a new database, but does not remove the old one (that can contain all kind of data).

Reply:
Added in https://github.com/City-of-Helsinki/kukkuu/pull/398